src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,53 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        Manufacturer audiManufacturer = new Manufacturer("Audi", "Germany");
+        Manufacturer infinityManufacturer = new Manufacturer("Infinity", "Japan");
+        Manufacturer teslaManufacturer = new Manufacturer("Tesla", "Tesla");
+
+        System.out.println(manufacturerService.create(audiManufacturer) + System.lineSeparator()
+                + manufacturerService.create(infinityManufacturer) + System.lineSeparator()
+                + manufacturerService.create(teslaManufacturer) + System.lineSeparator());
+
+        System.out.println(manufacturerService.get(16L) + System.lineSeparator()
+                + manufacturerService.get(999L));
```

Comment:
I can return null here, so the best way to show all info to avoid exception. I will do it through orElseThrow if the mentor says that what I wrote is bad.